### PR TITLE
Update ElasticsearchEngine.php

### PR DESCRIPTION
**Pull request by Julinar**
fix score order and pagination

### CODE REVIEW

File: src/ElasticsearchEngine.php [hunk 1]
```diff
@@ -169,6 +175,8 @@ protected function performSearch(Builder $builder, array $options = [])
             );
         }
 
+        //dd($this->elastic->search($params));
```
Comment (ErickTamayo) [inline src/ElasticsearchEngine.php]: Can you remove this please?

File: src/ElasticsearchEngine.php [hunk 2]
```diff
@@ -216,11 +224,21 @@ public function map(Builder $builder, $results, $model)
 
         $keys = collect($results['hits']['hits'])->pluck('_id')->values()->all();
 
-        return $model->getScoutModelsByIds(
+
+      /*  return $model->getScoutModelsByIds(
```
Comment (ErickTamayo) [inline src/ElasticsearchEngine.php]: Remove this comment as well

**Review by ErickTamayo — CHANGES_REQUESTED** [review]

**Comment by ErickTamayo** [comment]
Can you make the test pass again so I can merge this?

File: src/ElasticsearchEngine.php (hunk 1 above)
Comment (Julinar) [reply src/ElasticsearchEngine.php]: it's removed

File: src/ElasticsearchEngine.php (hunk 2 above)
Comment (Julinar) [reply src/ElasticsearchEngine.php]: same

**Comment by ErickTamayo** [comment]
Sadly this is still failing. I cannot merge until the test is fixed

**Comment by Julinar** [comment]
> Sadly this is still failing. I cannot merge until the test is fixed

Well tbh it's my first contrib on github and I didnt know why the test fail.
I checked on my branch and the return is :

```
Array
(
    [index] => scout
    [type] => table
    [body] => Array
        (
            [query] => Array
                (
                    [bool] => Array
                        (
                            [must] => Array
                                (
                                    [0] => Array
                                        (
                                            [query_string] => Array
                                                (
                                                    [query] => *zonda*
                                                )
                                        )
                                    [1] => Array
                                        (
                                            [match_phrase] => Array
                                                (
                                                    [foo] => 1
                                                )
                                        )
                                    [2] => Array
                                        (
                                            [terms] => Array
                                                (
                                                    [bar] => Array
                                                        (
                                                            [0] => 1
                                                            [1] => 3
                                                        )
                                                )
                                        )
                                )
                        )
                )
            [track_scores] => 1
            [sort] => Array
                (
                    [id] => Array
                        (
                            [order] => desc
                        )
                )
        )
)
```

for the test : 

```
    public function test_search_sends_correct_parameters_to_elasticsearch()
    {
        $client = Mockery::mock('Elasticsearch\Client');
        $client->shouldReceive('search')->with([
            'index' => 'scout',
            'type' => 'table',
            'body' => [
                'query' => [
                    'bool' => [
                        'must' => [
                            ['query_string' => ['query' => '*zonda*']],
                            ['match_phrase' => ['foo' => 1]],
                            ['terms' => ['bar' => [1, 3]]],
                        ]
                    ], 
                ],
                'track_scores' => 1,
                'sort' => [
                    ['id' => ['order' => 'desc']]
                ]
            ]
        ]);
```
test error log : 

```
There was 1 error:
1) ElasticsearchEngineTest::test_search_sends_correct_parameters_to_elasticsearch
Mockery\Exception\NoMatchingExpectationException: No matching handler found for Mockery_0_Elasticsearch_Client::search(array('index'=>'scout','type'=>'table','body'=>'array(...)',)). Either the method was unexpected or its arguments matched no expected argument list for this method
```

If you have an idea of why test test fail I take it :)

**Pull request closed without merging**